### DISCUSSION TITLE
Installing old version

### DISCUSSION THREAD

**Discussion by FelipeSBarros** [semiautomaticgit/SemiAutomaticClassificationPlugin#341] [Q&A]
Dear, Luca @semiautomaticgit 
for pedagogical purpose I have decided to tech my studente the old version of SCP (version 7).
I have tryied intalling it reading [this post](); The registration of the repository went well.
But after installing I got the following error:

> El complemento ha desaparecido: The plugin seems to have been installed but it's not possible to know where. The directory "/home/felipe/.local/share/QGIS/QGIS3/profiles/default/python/plugins/SemiAutomaticClassificationPlugin-master" has not been found. Probably the plugin package contained a wrong named directory. Please search the list of installed plugins. You should find the plugin there, but it's not possible to determine which of them it is and it's also not possible to inform you about available updates. Please contact the plugin author and submit this issue.

Looking to the folder indicated I could find the following folder `SemiAutomaticClassificationPlugin_v7-master`.
So I wonder, How could I "add" (o register) this folder as the folder of the installed SCP.

Of course, before installing the old version I have uninstalled the official one.
Would you be able to help?
Best regards

Felipe

**Comment by FelipeSBarros** [comment]
For now I have change the folder's name to match the one mentioned in the error mesage.
Is there another way to solve this problem?
Thanks
Felipe

**Comment by semiautomaticgit** [comment]
> For now I have change the folder's name to match the one mentioned in the error mesage

No need to rename.
You should simply restart QGIS.
You can also install version 7 along with version 8.

**Reply by FelipeSBarros** [reply]
> You should simply restart QGIS.

I have thried restarting QGIS, didn't work.
Thanks

**Reply by semiautomaticgit** [reply]
I suggest to uninstall any version of SCP then follow this guide and restart QGIS
https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#can-i-use-the-previous-version-7-of-scp
